### DISCUSSION TITLE
eqMac v1.3.2 - Minor emergency bug fix for Bluetooth devices

### DISCUSSION THREAD

**Discussion by nodeful** [bitgapp/eqMac#597] [General]
* Fixed volume being stuck at certain values
* Cmd + W Hide window now works when not using Dock mode

<hr /><em>This discussion was created from the release <a href='https://github.com/bitgapp/eqMac/releases/tag/'>eqMac v1.3.2 - Minor emergency bug fix for Bluetooth devices</a>.</em>

**Comment by bodhiwd** [comment]
@nodeful this isn't being picked up by my "Check for updates"-button in my eqMac app, and the main github page (https://github.com/bitgapp/eqMac) is still listing 1.3.0 being the latest release in the sidebar:

![Screenshot 2021-11-22 at 10 25 38](https://user-images.githubusercontent.com/48051498/142828007-69f43c8c-2526-4c8c-9bf5-65c0fa16eba0.png)

Not sure if these are related.

**Reply by nodeful** [reply]
Yes this was a Pre-release, I was waiting for more people to give me some feedback and I wanted to test this build on some older macOS VM's like 10.12, which I did today and confirmed that it works fine, so now I'm marking this release as public :) Thanks for reminding me though!
